INSTALLER.md

```diff
@@ -91,8 +98,67 @@ For code signing to work, you will need to set the `HF_PFX_FILE` and `HF_PFX_PAS
 1.  Perform a Release build of `package`
       Installer is now available in `build/_CPack_Packages/Darwin/DragNDrop
       
-### FAQ
-
-1. **Problem:** Failure to open a file. ```File: failed opening file "\FOLDERSHARE\XYZSRelease\...\Credits.rtf" Error in script "C:\TFS\XYZProject\Releases\NullsoftInstaller\XYZWin7Installer.nsi" on line 77 -- aborting creation process```
-    1. **Cause:** The complete path (current directory + relative path) has to be < 260 characters to any of the relevant files.
-    1. **Solution:** Move your build and packaging folder as high up in the drive as possible to prevent an overage.
+### Linux
+
+#### Server
+
+##### Ubuntu | .deb
+
+1. Ensure you are using an Ubuntu 18.04 system.
+2. Get and bootstrap Vircadia Builder.
+    ```
+    git clone https://github.com/vircadia/vircadia-builder.git
+    cd vircadia-builder
+    chmod +x vircadia-builder
+    ```
+3. Set up Vircadia Builder to compile the server. See [here](BUILD.md#possible-environment-variables) for possible environment variables and settings.
+    ```
+    ./vircadia-builder --build server
+    ```
+4. Build the server.
+    ```
+    ./vircadia-builder --build server
+    ```
+5. Navigate to the `pkg-scripts` directory.
+    ```
+    cd ../Vircadia/source/pkg-scripts/
+    ```
+6. Generate the .deb package. Set `DEBVERSION` to a valid semantic version value e.g. "2021.1.0".
+    ```
+    DEBVERSION="Semver e.g. 2021.1.0" DEBEMAIL="your-email@somewhere.com" DEBFULLNAME="Your Full Name" ./make-deb-server
+    ```
+7. If successful, the generated .deb package will be in the `pkg-scripts` folder.
+
+##### Amazon Linux 2 | .rpm
+
+1. Ensure you are using an Amazon Linux 2 system.
+2. Update the system and install dependencies.
+    ```
+    sudo yum update -y
+    sudo yum install git -y
+    sudo yum install rpm-build
+    ```
+3. Get and bootstrap Vircadia Builder.
+    ```
+    git clone https://github.com/vircadia/vircadia-builder.git
+    cd vircadia-builder
+    sudo ./install_amazon_linux_deps.sh
+    chmod +x vircadia-builder
+    ```
+4. Set up Vircadia Builder to compile the server. See [here](BUILD.md#possible-environment-variables) for possible environment variables and settings.
```

Comment:
Same as above:
You are not compiling the server here and building it in the next step.
```suggestion
4. Run Vircadia Builder to build the server. See [here](BUILD.md#possible-environment-variables) for possible environment variables and settings.
```